test/unit/phys/dca_step/cluster_solver/shared_tools/accumulation/single_sector_accumulation_test.hpp

```diff
@@ -152,20 +152,18 @@ void SingleSectorAccumulationTest<n_bands, n_sites, n_frqs>::prepareConfiguratio
     Configuration& config, Matrix& M, const int nb, const int nr, const double beta, const int n) {
   config.resize(n);
   M.resize(n);
-  dca::math::random::StdRandomWrapper<std::ranlux48_base> rng(0, 1, 0);
+  static dca::math::random::StdRandomWrapper<std::ranlux48_base> rng(0, 1, 0);
```

Comment:
Why make this static?

Reply:
To use the same random numbers as the test of the old code, as it used only one function call.